ranger/data/scope.sh

```diff
@@ -468,7 +468,8 @@ handle_fallback() {
 
 MIMETYPE="$( file --dereference --brief --mime-type -- "${FILE_PATH}" )"
 if [[ "${PV_IMAGE_ENABLED}" == 'True' ]]; then
-    handle_image "${MIMETYPE}"
+    # don't generate thumbnail for incomplete torrent video file
+    [[ "$MIMETYPE" =~ video/* && "$FILE_EXTENSION_LOWER" = "part" ]] || handle_image "${MIMETYPE}"
```

Comment:
I added the check here, now it's showing text as preview.

Reply:
I don't like this, it's not in the correct spot. Things related to a case should exclusively go inside the case.

Let's look into why the break wasn't working in the case instead.
The reason you were still seeing a preview is probably that the preview was cached?